### ISSUE TITLE
Module autosummary not showing

### ISSUE THREAD

**Issue by joelberkeley-pio** [readthedocs/sphinx-autoapi#225]
I'd like to show autosummary tables using sphinx-autoapi, and I'm using the `"show-module-summary"` option as
```
autoapi_add_toctree_entry = False
autoapi_python_class_content = "both"
autoapi_options = [
    'members',
    'show-inheritance',
    'show-module-summary',
    'special-members',
    'imported-members',
    'inherited-members',
]
```
but no tables are appearing in the built docs. I assume I should be seeing a table of contents for each package or module, with each item accompanied by (some of) the docstring for that item.

I'm using the "pydata_sphinx_theme" theme

**Comment by AWhetter** [comment]
Which version are you using? This should have been fixed in v1.4.0.

**Comment by joelberkeley-pio** [comment]
ah thanks, yes I was on 1.3. It's showing now
